<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WikeNotes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="searchBox">
      <property name="toolTip">
       <string>Type whatever you want to find here</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Matched in</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Everywhere</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Title and Content</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Content only</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Title only</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tag only</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QSplitter" name="vsplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListView" name="tagView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <widget class="NoteList" name="noteList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>0</width>
          <height>0</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Library"/>
    <addaction name="action_Open_Notes_Library"/>
    <addaction name="separator"/>
    <addaction name="action_New_Plain_Text_Note"/>
    <addaction name="action_New_Note"/>
    <addaction name="action_Edit_Selected_Note"/>
    <addaction name="action_Save_Note"/>
    <addaction name="action_Delete_Selected_Note"/>
    <addaction name="separator"/>
    <addaction name="action_Import"/>
    <addaction name="action_Export_Notes"/>
    <addaction name="separator"/>
    <addaction name="actionE_Xit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menu_Language">
     <property name="title">
      <string>&amp;Language</string>
     </property>
    </widget>
    <addaction name="actionText_Note_Font"/>
    <addaction name="action_Hotkey_Settings"/>
    <addaction name="menu_Language"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionUsage"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open_Notes_Library"/>
   <addaction name="action_New_Plain_Text_Note"/>
   <addaction name="action_New_Note"/>
   <addaction name="action_Edit_Selected_Note"/>
   <addaction name="action_Save_Note"/>
   <addaction name="action_Delete_Selected_Note"/>
   <addaction name="actionUsage"/>
   <addaction name="action_About"/>
  </widget>
  <action name="actionE_Xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_New_Note">
   <property name="icon">
    <iconset resource="resources/wike.qrc">
     <normaloff>:/new_text.png</normaloff>
     <normalon>:/new.png</normalon>:/new_text.png</iconset>
   </property>
   <property name="text">
    <string>New &amp;HTML Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Edit_Selected_Note">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Edit Selected Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Import">
   <property name="text">
    <string>&amp;Import Notes</string>
   </property>
  </action>
  <action name="actionText_Note_Font">
   <property name="text">
    <string>Text Note &amp;Font</string>
   </property>
  </action>
  <action name="action_Delete_Selected_Note">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/delete.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Delete Selected Note</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_Save_Note">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Export_Notes">
   <property name="text">
    <string>&amp;Export Notes</string>
   </property>
  </action>
  <action name="action_Hotkey_Settings">
   <property name="text">
    <string>&amp;Hotkey Settings</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset>
     <normalon>:/about.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="icon">
    <iconset>
     <normalon>:/help.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Usage</string>
   </property>
  </action>
  <action name="action_Open_Notes_Library">
   <property name="icon">
    <iconset>
     <normalon>:/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open Notes Library</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew_Library">
   <property name="text">
    <string>New &amp;Library</string>
   </property>
  </action>
  <action name="actionFocusSearchBox">
   <property name="text">
    <string>focusSearchBox</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_New_Plain_Text_Note">
   <property name="icon">
    <iconset resource="resources/wike.qrc">
     <normaloff>:/new.png</normaloff>:/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Plain Text Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NoteList</class>
   <extends>QWidget</extends>
   <header>notelist.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/wike.qrc"/>
 </resources>
 <connections/>
</ui>
